discovery-service/src/main/java/com/ca/mfaas/discovery/staticdef/ServiceDefinitionProcessor.java

```diff
@@ -86,7 +91,7 @@
         Map<String, String> ymlSources = new HashMap<>();
 
         if (ymlFiles == null) {
-            log.error("I/O problem occurred during reading directory: {}", directory.getAbsolutePath());
```

Comment:
Shouldn't we log when we can't read directory? There is also log.info bellow that ...

Reply:
there is a debug log on line 94 for this purpose. From the nature of the line it feels to us that log.debug is appropriate here. Log info is not a problem, I suggest you review the new filter setup first to understand why

Reply:
I have added it, tests with invalid or missing directories give 2019-10-15 07:55:32.275 <ZWEADS1:Thread-47:11556> janda06 WARN  (c.c.m.d.s.ServiceDefinitionProcessor) ZWEAD700W Static Api definition directory 'brekeke' is not a directory or does not exist {1f5b0078-12f6-4ed0-be7c-e423e5064898}. I think this log would happen very rarely.

Reply:
I suppose this could happen when a server does not have sufficient access rights for the directory. Let's hope this will be happening rarely ...

Reply:
btw. there is still  ... log.info("No static service definition found in directory: {}", directory.getAbsolutePath());

Reply:
@Jirka check LogLevelInfoFilter class

Reply:
ok, so why not simply use log.debug? Why should info messages behave like debug?

Reply:
because content of those messages is not debug. This is from clear code perspective. They are info messages and even if you turn on debug, they should say [INFO] level. The advantage is that you are not losing the distinction. 
I think that if you take log.info and replace it with log.debug, you are in fact making the log messages into debug.

Reply:
Ok, I still don't see much difference between them.